src/app/lib/config/services/afrique.js

```diff
@@ -7,7 +7,7 @@ const service = {
   default: {
     lang: `fr`,
     articleAuthor: `https://facebook.com/bbcnewsafrique`,
-    articleTimestampPrefix: 'Updated',
+    articleTimestampPrefix: 'Mise à jour ',
```

Comment:
These usually don't have a space after them.
```suggestion
    articleTimestampPrefix: 'Mise à jour',
```